compiler/testData/diagnostics/tests/modifiers/IllegalModifiers.fir.kt

```diff
@@ -7,14 +7,14 @@ abstract class A() {
     <!INCOMPATIBLE_MODIFIERS!>final<!> <!INCOMPATIBLE_MODIFIERS!>open<!> fun h() {}
 
     open var r: String
-    get
-    abstract protected set
+        get
```

Comment:
There are many unnecessary changes regarding spaces/intents. Are those test comparison-generated or accidentally added?

Reply:
It's generated by the test infra.